BigBaseV2/src/views/players/view_players.cpp

```diff
@@ -10,7 +10,9 @@ namespace big
 		if (plyr->is_host())
 			ImGui::PushStyleColor(ImGuiCol_Button, ImVec4(0.29f, 0.45f, 0.69f, 1.f));
 
-		if (ImGui::Button(plyr->get_name(), { ImGui::GetWindowSize().x - 15.f, 0.f }))
+		std::string btnLbl = std::string(plyr->get_name()) + "##player" + std::to_string(plyr->id());
```

Comment:
This can be replaced with the following code and is most likely more efficient than converting from int to string and then  doing string concatenation.
```cpp
ImGui::PushID(plyr->id());
if (ImGui::Button(plyr->get_name(), {ImGui::GetWindowSize().x - 15.f, 0.f}))
{
	g_player_service->set_selected(plyr);
	g_gui_service->set_selected(tabs::PLAYER);
	g->window.switched_view = true;
}
ImGui::PopID();
```

I have the modified code ready to push so if you want I can quickly change this.

Reply:
Sure, thanks!